Firestore/Example/Tests/Integration/API/FIRDatabaseTests.mm

```diff
@@ -1813,4 +1815,10 @@ - (void)testMinimumCacheSize {
                           NSException);
 }
 
+- (void)testUnlimitedCacheSize {
+  FIRPersistentCacheSettings *settings =
+      [[FIRPersistentCacheSettings alloc] initWithSizeBytes:@(Settings::CacheSizeUnlimited)];
+  XCTAssertEqual(settings.internalSettings.size_bytes(), Settings::CacheSizeUnlimited);
```

Comment:
Can we try actually initialize the SDK with this settings object and do something (anything)?